jms-source/README.didact.md

```diff
@@ -67,66 +63,45 @@ oc create configmap jms-source-config --from-file jms-source/configs/application
 
-This example is based on [Apache Camel's JMS](https://camel.apache.org/components/latest/jms-component.html) component. As such we can use the Spring Boot auto-configuration for convenience and adjust the properties `camel.component.jms.connection-factory` to set the connection factory.
-
-Setting the message broker address is specific to the JMS client being used. When using the AMQP 1.0 protocol with the Apache Qpid JMS client, this is done through the property `camel.component.jms.connection-factory.remoteURI`. When using the Apache Artemis CORE protocol, this is done via the property `camel.component.jms.connection-factory.brokerURL`.
-
-*Note*: the configuration file contains commented examples for both properties.
+*Note*: you must edit the file and provide your broker host and port.
 
 The second set of parameters that may need to be adjusted are the `destination type`, which is used to inform whether a `queue` or a `topic` will be used and the destination name. These two configurations are referenced in the component configuration (i.e: using `{{jms.destinationType}}` and `{{jms.destinationName}}` respectively).
 
-
 ## Understanding the Example
 
 The example generates consumes messages published on to the message broker. To understand the example, please access the [source code](JmsSourceExample.java).
 
-
 ## Runtime Considerations
 
+The JMS is a flexible standard, therefore, users are free to choose the message broker, protocol and clients that are suitable for their solution. This example demonstrates how to use the Apache QPid JMS client with the client dependency: `mvn:org.amqphub.quarkus:quarkus-qpid-jms`.
 
-The JMS is a flexible standard, therefore, users are free to choose the message broker, protocol and clients that are suitable for their solution.
-
-This example demonstrates how the Apache Artemis Core Client or the Apache QPid JMS client can be used to access a compatible message broker speaking the Apache Artemis Core protocol (such as Apache Artemis) or AMQP 1.0 (such as Apache ActiveMQ, Apache Artemis, Azure Service Bus and others).
-
-To use this example with the Apache Artemis Core Client, the respective client dependency must be informed (i.e.: `mvn:org.apache.activemq:artemis-jms-client-all:2.17.0`).
-
-To use this example with the Apache QPid JMS client, the following client dependency is used: `mvn:org.apache.qpid:qpid-jms-client:jar:1.0.0`.
-
-
-## Running the Example: Using Apache Artemis Core Client
-
-
-To run the project using the Apache Artemis Core Client, use:
+## Running the Example
 
+To run the project you can use:
 
 ```
-kamel run --configmap jms-source-config -d mvn:org.apache.activemq:artemis-jms-client-all:2.17.0 --dev JmsSourceExample.java
+kamel run --config configmap:jms-source-config -d mvn:org.amqphub.quarkus:quarkus-qpid-jms --dev JmsSourceExample.java
```

Comment:
Are we sure this one will work on the productized Camel K? I'm under the impression the format `configmap:name` only works on 1.5 ... no?

Reply:
it is Comunity 1.5 and productized 1.4
@squakez implemented it and backported it :-)

Reply:
That's awesome!